src/scripts/main.js

```diff
@@ -354,7 +354,32 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const createdTr = document.createElement('tr');
+  const { sex, born, died } = person;
+  const tableRowData = {
+    name: person.name,
+    gender: sex,
+    born,
+    died,
+    age: died - born,
+    century: Math.ceil(died / 100),
+  };
 
-// write your code here
+  if (sex === 'm') {
+    tableRowData.gender = 'Male';
+  } else {
+    tableRowData.gender = 'Female';
+  }
```

Comment:
You can simplify it like this 
```suggestion
  tableRowData.gender = sex === 'm' ? 'Male' : 'Female'
```